docs/src/man/getting_started.md

```diff
@@ -785,6 +785,56 @@ not be mutated without caution.
 The exact rules of handling columns of a `DataFrame` are explained in
 [The design of handling of columns of a `DataFrame`](@ref man-columnhandling) section of the manual.
 
+## Replacing Data
+
+Use the broadcasting syntax to apply replacement operations to data frames.
```

Comment:
It would be useful to specify that this is to replace values in all columns of a data frame. While you're at it, can you show an example replacing values in a single column? Two solutions are possible: using `ifelse` or using `replace!` (which is a bit shorter).